src/docs/components/properties.md

```diff
@@ -312,9 +312,46 @@ export class MyHttpService {
 
 In TypeScript, `MyHttpService` is both an `Object` and a 'type'. When using user-defined types like `MyHttpService`, the
 type must always be exported using the `export` keyword where it is declared. The reason for this is Stencil needs to
-know what type the prop `myHttpService` is when passing an instance of `MyHttpService` to `TodoList` from a parent
+know what type the prop `httpService` is when passing an instance of `MyHttpService` to `TodoListItem` from a parent
 component.
 
+To set `httpService` in TSX, assign the prop name in the custom element's tag to the desired value like so:
```

Comment:
I'm not sure on this one - I don't think we necessarily need to call out non-primitives here, since we're in the `Object` section of the docs (and cover those types prior to this section).

If I remove that section calling out types of the class member decorated with `@Prop()`, we get:
```
Once we've used the @Prop() decorator to define a custom attribute, we can pass a value into our component in JSX within another Stencil component:
```

Which I think is roughly equivalent to what we have now? I do want to expand ['prop name' a bit to prevent muddying the differences between a property and an attribute](7658581d8) further though. LMK what you think!